ensemble_predict.py

```
@@ -0,0 +1,123 @@
+
+        try:
+            # Kép betöltése és előkészítése (Ugyanúgy, mint a tanításnál!)
+            img = Image.open(image_path).convert('L')
+            img = img.resize((IMG_SIZE, IMG_SIZE))
+            img_array = np.array(img)
+            img_array = img_array / 255.0  # Normalizálás
+
+            # (1, 32, 32, 1) formátum
+            img_ready = img_array.reshape(1, IMG_SIZE, IMG_SIZE, 1)
+
+        except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)